ci/flip-production-traffic.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -u
+
+function log {
+   echo "$(date +"%T") - INFO - $*"
+}
+
+function read_version () {
+    CLUSTER=$1
```

Comment:
Why `CLUSTER` ? In App Engine you work with _projects_

Reply:
Copy and paste legacy ...